frontend/src/concepts/pipelines/content/createRun/contentSections/RunTypeSection.tsx

```diff
@@ -29,7 +33,12 @@ export const RunTypeSection: React.FC<RunTypeSectionProps> = ({ data, isDuplicat
       To create a schedule that executes recurring runs,{' '}
       <Link
         to={createRecurringRunRoute(namespace, experiment?.experiment_id)}
-        state={{ locationData: data }}
+        state={{
+          locationData: data,
+          contextData: {
+            versionToUse: PipelineVersionToUse.LATEST,
+          },
```

Comment:
This is not correct, because we need to carry over what we set for the pipeline version on the create-run page to the create-schedule page. If we are doing this, when switching between the 2 pages, we will lose the context of what pipeline version the user chose before and always select the latest radio. I think we could remove the `contextData` here because it's only used to create run/schedules from a specific version/experiment. The `locationData` contains everything we have including the pipeline version, we should rely on that and always set to `PipelineVersionToUse.PROVIDED` if the `locationVersion` is set.